MANIFEST.in

```diff
@@ -1,7 +1,12 @@
+include *.csv
+include *.md
+include *.yml
+include CODEOWNERS
 include container-apigentools
-include Dockerfile
 include docker-entrypoint.sh
+include Dockerfile
 include LICENSE
 include README.md
-include requirements.txt
-include requirements-tests.txt
+recursive-include docs *.md
+recursive-include tests *.json
+recursive-include tests *.py
```

Comment:
Why include all the docs and tests in there ?

Reply:
Do we want to distribute tests and docs as part of the zip file. Otherwise I change it to `prune docs` and `prune tests`.